be/src/runtime/data_stream_common.h

```diff
@@ -0,0 +1,58 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+#include <map>
+
+#include "column/column_hash.h"
+#include "column/vectorized_fwd.h"
+#include "gen_cpp/Types_types.h" // for TUniqueId
+#include "runtime/descriptors.h" // for PlanNodeId
+
+namespace starrocks {
+
+// To manage pass through chunks between sink/sources in the same process.
+using ChunkUniquePtrVector = std::vector<vectorized::ChunkUniquePtr>;
```

Comment:
This h file should rename to `local_data_stream_manager` or `local_pass_through_nanager`.

Reply:
OK